src/SecureStore.BeyondTrust/BeyondTrustVaultClientFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace UiPath.Orchestrator.Extensions.SecureStores.BeyondTrust
+{
+    public class BeyondTrustVaultClientFactory
+    {
+        private static BeyondTrustVaultClient client;
```

Comment:
You can't cache a single static client for the lifetime of the app. That means you can only configure a single credential store of this type per application (not per tenant), which is not a restriction we currently have. Also, caching a static instance means changing the configuration will have no effect until you restart the app.
See the Hashicorp implementation, which does caching of client per context: https://github.com/UiPath/Orchestrator-CredentialStorePlugins/blob/master/src/SecureStore.HashicorpVault/HashicorpVaultClientFactory.cs#L25